ansible/play

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+if [ -t 0 -a -t 1 -a -t 2 ]; then
+  uname=`uname -s`
+  if [ "$uname" = Linux ]; then
+    if [ -z "$BASH_VERSION" ]; then
+      exec bash "$0" "$@"
+    else
+      # I need some internal bash `printf` supporting %q
+      name=$(echo "play" "$@" | sed s/[^-+=a-zA-Z0-9_]/_/g).$(date +%Y.%m.%d.%H.%M.%S).$$.$(tput cols)x$(tput lines)
+      args=$(printf "%q " "$@")
+      exec /usr/bin/script --append --return --timing="./${name}.timing" -c "ansible-playbook --vault-password-file ./password-pipe --inventory-file ./inventory $args" "./${name}.log"
+    fi
+  elif [ "$uname" = Darwin ]; then
+    name=`echo "play" "$@" | sed s/[^-+=a-zA-Z0-9_]/_/g`.`date +%Y.%m.%d.%H.%M.%S`.$$.log
+    exec script -a "$name" ansible-playbook --vault-password-file ./password-pipe --inventory-file ./inventory "$@"
+  else
+    echo "$0: don't know how to launch \`script\` at $uname." 1>&2
+    # fall-through
+  fi
+fi
+exec ansible-playbook --vault-password-file ./password-pipe --inventory-file ./inventory "$@"
```

Comment:
Shouldn't `./inventory` -> `./ansible/inventory` and `./password-pipe` -> `./ansible/password-pipe` in this file?